src/test/java/de/tum/in/www1/artemis/connectors/AeolusTest.java

```diff
@@ -155,4 +155,17 @@ void testAeolusRepository() {
         aeolusRepository.setPath("path");
         assertThat(aeolusRepository.getPath()).isEqualTo("path");
     }
+
+    @Test
+    void testImageTagCombinations() {
+        DockerConfig dockerConfig = new DockerConfig();
+        dockerConfig.setImage("image");
+        dockerConfig.setTag("tag");
+        assertThat(dockerConfig.getFullImageName()).isEqualTo("image:tag");
+        dockerConfig.setTag(null);
+        assertThat(dockerConfig.getFullImageName()).isEqualTo("image:latest");
+        dockerConfig.setImage("image:tag");
+        dockerConfig.setTag("notshown");
+        assertThat(dockerConfig.getFullImageName()).isEqualTo("image:tag");
+    }
```

Comment:
Please separate the cases like this:
```suggestion
    void testImageTagCombinations() {
        DockerConfig dockerConfig = new DockerConfig();
        dockerConfig.setImage("image");
        dockerConfig.setTag("tag");
        assertThat(dockerConfig.getFullImageName()).isEqualTo("image:tag");
        
        dockerConfig.setTag(null);
        assertThat(dockerConfig.getFullImageName()).isEqualTo("image:latest");
        
        dockerConfig.setImage("image:tag");
        dockerConfig.setTag("notshown");
        assertThat(dockerConfig.getFullImageName()).isEqualTo("image:tag");
    }
```

Reply:
will do!